src/inboxsdk.d.ts

```diff
@@ -415,6 +415,13 @@ export interface DrawerDescriptor {
 export { type MoleOptions as MoleDescriptor };
 export interface ButtonDescriptor {
   title?: string;
+  /**
+   * The innerText of the button.
+   * If you want to use an icon font with ligatures enabled, you can use the `iconLigaDefault` property.
+   *
+   * @default '&nbsp;'
+   */
+  iconLigaDefault?: string;
```

Comment:
what's the reasoning for having "default" be part of this property name? it's not really a default, it's just setting something, so it was a bit confusing to me

Reply:
I'm not attached to the name by any means. Would this be more your speed?

```suggestion
  iconLiga?: string;
```

The TSDoc could be clearer that we only set this property if `iconClass` is also passed.